client/test/execution.test.js

```diff
@@ -417,6 +381,9 @@ describe('Execution', function() {
       it('should also populate the timeline with those events', async function() {
         this.retries(3) // flakey on mocha-chrome but not normal, windowed Chrome
         var [timelineEl] = await compactViewTest(this.test)
+
+        await Promise.delay(50)
```

Comment:
Instead of using a Promise.delay for 50ms, can you use jest mock timers andexecute a nextTick or setTimeout with no parameters to get the timeouts to resolve?

Reply:
Sorry I didn't update this one but will keep in mind for next time. I think most of the code there is using the above so I chose to just reuse what was already there and working.